JSDemos/Demos/PivotGrid/ExcelJSCellCustomization/description.md

```diff
@@ -1,9 +1,9 @@
-The **PivotGrid** allows you to&nbsp;customize data when exporting it&nbsp;to&nbsp;an&nbsp;Excel file: adjust the font options, specify a&nbsp;cell&rsquo;s background, modify exported values or&nbsp;specify their alignment and formatting options.
+You can use the [customizeCell](/Documentation/ApiReference/Common/Object_Structures/ExportPivotGridProps/#customizeCell) callback to modify cell data and formatting when you export PivotGrid to an Excel file. You can access and change the following attributes:
 
 - <a href="https://github.com/exceljs/exceljs#fonts" target="_blank">Font</a> options    
 - <a href="https://github.com/exceljs/exceljs#fills" target="_blank">Cell background</a>     
 - <a href="https://github.com/exceljs/exceljs#hyperlink-value" target="_blank">Cell values</a>       
 - Text <a href="https://github.com/exceljs/exceljs#alignment" target="_blank">alignment</a>        
 - <a href="https://github.com/exceljs/exceljs#number-formats" target="_blank">Formatting</a> options    
 
-The [customizeCell](/Documentation/ApiReference/Common/Object_Structures/ExportPivotGridProps/#customizeCell) function also allows you to identify cell types. For example, this demo changes the background color and font weight for cells depending of it's [area](/Documentation/ApiReference/UI_Widgets/dxPivotGrid/Configuration/#dataFieldArea).
\ No newline at end of file
+The **customizeCell** function also allows you to identify cell area types. For example, this demo shows how to change the background color and font weight for cells that belong to the certain [areas](/Documentation/ApiReference/UI_Widgets/dxPivotGrid/Configuration/#dataFieldArea).
```

Comment:
Maybe you can be more specific here? Instead of "certain areas" you can actually say which areas. It would be easier to locate that place in code and see what happens in the demo.